code/nodes/modelGeneration/tests/test.t

```diff
@@ -1 +1,124 @@
+configNormalClass:startDateTime,`featExtractType`problemType!`normal`class
+
+// list of input configurations
+configList:(configNLPReg;configNLPClass;configFRESHReg;
+            configFRESHClass;configNormalReg;configNormalClass)
+
+-1"\nTesting inappropriate problem type to build models";
+
+// Inappropriate config problem type
+inapprConfig:configNormalClass,enlist[`problemType]!enlist`failTest
+
+// inappropriate file error
+fileError:"text file not found"
+
+// Testing inappropriate problem type
+failingTest[.automl.modelGeneration.filesChk;inapprConfig;1b;fileError];
+
+-1"\nTesting appropriate problem type to build models";
+
+// Testing all appropriate problem types
+all passingTest[.automl.modelGeneration.filesChk;;1b;(::)]each configList
+
+-1"\nTesting appropriate inputs to extract desired model dictionary";
+
+// Appropriate configurations should return a dictionary
+parsePathGen:.automl.modelGeneration.txtParse[;"/code/customization/models/modelConfig/"]
+all 99h=/:type each parsePathGen each configList
+
+-1"\nTesting appropriate input to extract correct models based on problem type";
+
+// generate model dictionaries 
+modelDict     :.automl.modelGeneration.txtParse[;"/code/customization/models/modelConfig/"]
+regModelDict  :modelDict configNormalReg
+classModelDict:modelDict configNormalClass
+
+// target values
+tgtReg       :100?1f
+tgtClass     :100?0b
+tgtMultiClass:100?3
+
+// Testing that all problem types return a dictionary
+98h~type .automl.modelGeneration.modelPrep[configNormalReg  ;regModelDict  ;tgtReg       ]
+98h~type .automl.modelGeneration.modelPrep[configNormalClass;classModelDict;tgtClass     ]
+98h~type .automl.modelGeneration.modelPrep[configNormalClass;classModelDict;tgtMultiClass]
+
+-1"\nTesting appropriate inputs to build up the model based on naming convention";
+
+// Generate model table
+regModelTab  :flip`model`lib`fnc`seed`typ!flip key[regModelDict  ],'value regModelDict
+classModelTab:flip`model`lib`fnc`seed`typ!flip key[classModelDict],'value classModelDict
+
+// Test that all inputs return a lambda or projection type
+all (type each .automl.modelGeneration.mdlFunc .'flip regModelTab`lib`fnc`model  )in 100 104h 
+all (type each .automl.modelGeneration.mdlFunc .'flip classModelTab`lib`fnc`model)in 100 104h
+
+-1"\nTesting appropriate target values to ensure the correct models are used";
+
+// target values that exceed the 10000 limit
+tgtRegLarge       :20000?1f
+tgtClassLarge     :20000?0b
+tgtMultiClassLarge:20000?3
+
+// Test all appropriate model tables and target values that are less than threshold
+passingTest[.automl.modelGeneration.updModels;(regModelTab  ;tgtReg       );0b;regModelTab  ]
+passingTest[.automl.modelGeneration.updModels;(classModelTab;tgtClass     );0b;classModelTab]
+passingTest[.automl.modelGeneration.updModels;(classModelTab;tgtMultiClass);0b;classModelTab]
+
+// Test all appropriate model tables and target values that are greater than threshold
```

Comment:
These tests should check that whatever models are expected to be removed in the case that a large number of targets are provided actually removes their presence i.e. 

```
regModelsUpd:.automl.modelGeneration.updModels[regModelTab  ;tgtRegLarge]
all null exec from regModelsUpd where mdlName=`notrelevant
```